pkgs/games/eduke32/default.nix

```diff
@@ -58,7 +58,7 @@ in stdenv.mkDerivation rec {
     done
   '';
 
-  NIX_CFLAGS_COMPILE = "-I${SDL2.dev}/include/SDL2 -I${SDL2_mixer}/include/SDL2";
+  NIX_CFLAGS_COMPILE = "-I${SDL2.dev}/include/SDL2 -I${lib.getDev SDL2_mixer}/include/SDL2";
```

Comment:
Maybe use lib.getDev for SDL2 too for consistency? Feels weird to have two ways of doing the same thing on the same line :)

Reply:
Sounds good. Updated to use `lib.getDev SDL2`.